spec/models/event_spec.rb

```diff
@@ -60,4 +60,30 @@
       expect { create(:event, eventable: eventable) }.to change { eventable.reload.updated_at }
     end
   end
+
+  describe '#for_feed' do
+    subject(:event) { create(:event) }
```

Comment:
```suggestion
```

Reply:
We can remove it but discussed that it doesn't have a performance cost and makes the test more readable/more consistent with the other specs for the `#for_feed` tests